src/lib/pubkey/curve25519/curve25519.cpp

```diff
@@ -88,7 +88,7 @@ Curve25519_PrivateKey::Curve25519_PrivateKey(const AlgorithmIdentifier&,
    size_check(m_private.size(), "private key");
    }
 
-secure_vector<byte> Curve25519_PrivateKey::pkcs8_private_key() const
+secure_vector<byte> Curve25519_PrivateKey::private_key_bits() const
    {
    return DER_Encoder()
       .start_cons(SEQUENCE)
```

Comment:
There should be a comment here that what's being encoded is not an `ECPrivateKey` type at all but an ad-hoc botan specific format under a private OID. I suppose it would be possible to stick x25519 into ECPrivateKey but I don't know of any impl that transfers 25519 keys in that way.